extensions/spring-data-rest/runtime/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>quarkus-spring-data-rest-parent</artifactId>
+    <groupId>io.quarkus</groupId>
+    <version>999-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>quarkus-spring-data-rest</artifactId>
+  <name>Quarkus - Spring - Data REST - Runtime</name>
+  <description>Generate JAX-RS resources for a Spring Data application</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-rest-data-panache</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-spring-data-jpa</artifactId>
+    </dependency>
+    <dependency><!-- TODO create Quarkus Spring Data REST API jar -->
```

Comment:
So I understand this shouldn't be merged for now?

Reply:
@geoand what do you think? Should be merge it with a regular Spring dependency and replace it once we have a new minimal one? Or should we postpone merging until we have a minimal dependency ready?
We replaced them later in the other Spring extensions, no?

Reply:
Oh shoot...

I didn't notice this, although I did think about it at some point... My bad!

We should not include the upstream `spring-data-rest-core `. As now none of our Spring deps depend on upstream spring stuff, we should keep up with that pattern and create a new miminal jar before merging this.

Reply:
Sure, I'll create a new minimal JAR for Spring Data REST then.